src/transform/title.ts

```diff
@@ -6,11 +6,18 @@ export = function extractTitle(tokens: Token[]) {
         titleTokens: Token[] = [];
 
     if (Array.isArray(tokens) && tokens.length > 0) {
-        if (tokens[0].type === 'heading_open' && tokens[0].tag === 'h1') {
-            titleTokens = tokens[1].children || [];
-            title = tokens[1].content;
+        let start = 0;
+        const skipTokenType = ['__yfm_lint'];
+
+        while (skipTokenType.includes(tokens[start].type)) {
+            start++;
+        }
+
+        if (tokens[start].type === 'heading_open' && tokens[start].tag === 'h1') {
+            titleTokens = tokens[start + 1].children || [];
+            title = tokens[start + 1].content;
             // cut out "heading_open", "inline" and "heading_close" tokens
-            contentTokens = tokens.slice(3);
+            contentTokens = [...tokens.slice(0, start), ...tokens.slice(start + 3)];
```

Comment:
Who do you need this change?

Reply:
In term plugin I push the `__yfm_lint` token to define warnings. If the `__yfm_lint` token became the first token on the page, then function `extractTitle` does not work properly, because it expects the first token to be `heading_open`. So I skip all first lint tokens.

As the alternative, I thought that I can pass `runningInLint` variable in `pluginOptions` and based on that add or not `__yfm_lint` token. But this will lead to inconsistency tokens when run on lint and not.